Havit.Blazor.Components.Web.Bootstrap/Navigation/HxSidebarItem.razor.cs

```diff
@@ -133,6 +164,12 @@ protected virtual async ValueTask DisposeAsyncCore()
 		{
 			await ParentSidebarItem.ChildItemsRegistration.UnregisterAsync(this);
 		}
+
+		if (popperInitialized)
+		{
+			popperInitialized = false;
+			await jsModule.InvokeVoidAsync("destroyPopper", navId, id);
+		}
```

Comment:
Missing `jsModule.Dispose` + usual `try-catch` logic to handle Blazor Server scenarios (circuit closed before disposal logic finishes etc.)

Reply:
The `destroyPopper` method is meant to represent `dispose`. Should I rename it?